website/docs/r/dnat.html.markdown

```diff
@@ -44,7 +43,8 @@ resource "vcd_dnat" "forIcmp" {
 The following arguments are supported:
 
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the DNAT
-* `network_name` - (Optional; *v2.2+*) The name of the organization network on which to apply the DNAT. *`network_name` will be required field in the next major version.*
+* `network_type` - (Optional; *v2.2+*) Type of the network on which to apply the NAT rule. Default: "org". *`network_type` will be required field in the next major version.*
```

Comment:
"will be required field" -> "will be a required field"

Reply:
changed